lib/aggregation/accumulator/src/test/accumulator-config-test.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const { extend, omit } = require('underscore');
+const { createAccumulatorConfig } = require('../lib/accumulator-config');
+
+describe('accumulator reducer config test', () => {
+  const dedupId = 'dedup-id';
+
+  const udoc = {
+    organization_id: 'org-id',
+    resource_instance_id: 'res-instance-id',
+    consumer_id: 'consumer-id',
+    resource_id: 'resource-id',
+    plan_id: 'plan-id',
+    metering_plan_id: 'metering-plan-id',
+    rating_plan_id: 'rating-plan-id',
+    pricing_plan_id: 'pricing-plan-id'
+  };
+
+  let reducerConfig;
+  let sandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  const validateReducerConfig = (msg, isSecured) => context(msg, () => {
+    beforeEach(() => {
+      reducerConfig = createAccumulatorConfig(sandbox.stub().returns(isSecured), sandbox.any, sandbox.any);
```

Comment:
Why does `createAccumulatorConfig` take a funtion, which tells whether the system is secured or not? Is there anything dynamic in the `createAccumulatorConfig` that needs that?

Reply:
(P.S. This was already discussed in a previous review)